client/idrepo/enduser/pom.xml

```diff
@@ -269,7 +269,6 @@ under the License.
             <configuration>
               <jvmArguments>
                 -Dwicket.core.settings.general.configuration-type=development
-                -XX:HotswapAgent=fatjar
```

Comment:
Please re-add this option.

Reply:
Sorry, missed to re-addd before push

Reply:
...or just install https://github.com/TravaOpenJDK/trava-jdk-11-dcevm and use that with `-Pdebug`